Form-Controls/index.html

```diff
@@ -1,28 +1,86 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Product Selection Form</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
 
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
+    <main>
+      <section class="container">
+        <!--  Name and Last Name -->
+        <section class="names-row">
+          <section class="couple-column">
+            <label for="name">Name:</label>
+            <input type="text" id="name" name="name" required />
+          </section>
+          <section class="couple-column">
+            <label for="lastname">Lastname:</label>
+            <input type="text" id="lastname" name="lastname" required />
+          </section>
+        </section>
```

Comment:
It's fantastic to see that you're using semantic HTML elements! Kudos to you for following best practices in structuring your web pages.